src/commands/force/org/clone.ts

```diff
@@ -121,10 +121,10 @@ export class OrgCloneCommand extends SfCommand<SandboxProcessObject> {
       });
 
       const { sandboxReq, srcSandboxName } = await createSandboxRequest(true, flags.definitionfile, logger, varargs);
```

Comment:
If you get the types correct, `createSandboxRequest` the compiler will know.

"if true was passed for `isClone` (the first arg) then srcSandboxName will be a string"

That you need a non-null assertion below should tell you that the types are wrong.